scripts/subsribe.sh

```diff
@@ -1,9 +1,12 @@
 #!/usr/bin/env bash
 
+firehose_delivery_stream_arn=$(aws firehose describe-delivery-stream --delivery-stream-name ${1} | jq -r '.DeliveryStreamDescription.DeliveryStreamARN')
+cwl_to_kinesis_role_arn="arn:aws:iam::${ACCOUNT_ID}:role/${2}"
+
 # subscribe lambda to cloudwatch and all lambdas
-$PROJECT_ROOT/exporters/cwl_to_elk/add_log_group.sh "$CLOUDTRAIL_LOG_GROUP_NAME"
+processors/firehose_to_es_processor/add_log_group.sh '/aws/cloudtrail/audit-and-data-access' $firehose_delivery_stream_arn $cwl_to_kinesis_role_arn
```

Comment:
Can we reintroduce `CLOUDTRAIL_LOG_GROUP_NAME`?

Reply:
agreed, will update